app/components/Upload/ImageUpload.tsx

```diff
@@ -239,12 +239,7 @@ export default class ImageUpload extends Component<Props, State> {
     const { cropOpen, file, files } = this.state;
     const preview = file && file.preview;
     const accept: Accept = {
-      'image/jpeg': ['*'],
-      'image/png': ['*'],
-      'image/gif': ['*'],
-      'image/tif': ['*'],
-      'image/bmp': ['*'],
-      'image/avif': ['*'],
+      'image/*': ['.jpeg', '.jpg', '.png', '.gif', '.tif', '.bmp', '.avif'],
```

Comment:
Support for .jpg coming soon

Cutting edge webapp 🧑‍💻

Reply:
On the plus side - the current implementation doesn't block any filetypes as it doesn't work, so we already support .jpg 🤷